src/app/ApplicationTemplate.UWP/Views/Content/Authentication/LoginPage.xaml

```diff
@@ -13,21 +13,40 @@
 
 	<Grid Background="{StaticResource MaterialBackgroundBrush}">
 
+		<VisualStateManager.VisualStateGroups>
+			<VisualStateGroup>
+
+				<VisualState x:Name="BiggerFont">
+
+					<!-- This represents bigger phones than iPhone 5, SE -->
+					<VisualState.StateTriggers>
+						<AdaptiveTrigger MinWindowHeight="630" />
+					</VisualState.StateTriggers>
+
+					<!-- We change the font size to adapt the design for small and big phones -->
+					<VisualState.Setters>
+						<Setter Target="Login_Title.FontSize"
+								Value="60" />
+					</VisualState.Setters>
+				</VisualState>
+			</VisualStateGroup>
+		</VisualStateManager.VisualStateGroups>
+
 		<!-- Background -->
 		<Image Source="ms-appx:///Assets/Background_Mustaches_Light.png"
 			   Stretch="UniformToFill"
 			   HorizontalAlignment="Stretch"
 			   VerticalAlignment="Stretch" />
 
-		<ScrollViewer>
+		<ScrollViewer toolkit:VisibleBoundsPadding.PaddingMask="Top,Bottom">
```

Comment:
We avoid putting PaddingMask on the scrollviewer, let's put it on the stackpanel instead